sympy/utilities/lambdify.py

```diff
@@ -102,10 +102,10 @@
 NUMEXPR_TRANSLATIONS = {}
 
 # Available modules:
-MODULES = {
+_MODULES = {
```

Comment:
Yes, you're probably right. That dictionary is affected by calls to lambdify though so people really shouldn't be, but I'll revert the change.